components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -30,6 +30,9 @@
 from strenum import KebabCaseStrEnum
 
 # CONSTANTS
+DECOMPRESSION_COMMAND = "x"
```

Comment:
Was thinking about using a enum similar to JobType, but will need to manually assign a value instead of using auto.